src/app/plugins/cat.plugin.ts

```diff
@@ -81,6 +77,16 @@ export class CatPlugin extends Plugin {
       .catch((err) => this.container.loggerService.warn(err));
   }
 
+  private async _getListEmbed() {
+    const breedsAsArray = this._breeds.map((breedData: { name: string; id: string }) => {
+      return breedData.name;
+    });
+    this._embedBreeds = await this.container.messageService.generateEmbedList(
+      breedsAsArray,
+      new MessageEmbed()
```

Comment:
What's the point in passing in `new MessageEmbed()`?

Reply:
oops mb that was from testing